docs/sql-programming-guide.md

```diff
@@ -1776,6 +1776,66 @@ working with timestamps in `pandas_udf`s to get the best performance, see
 
 ## Upgrading From Spark SQL 2.2 to 2.3
 
+  - Since Spark 2.3, Spark supports a vectorized ORC reader with a new ORC file format for ORC files and Hive ORC tables. To do that, the following configurations are newly added or change their default values.
+
+    <table class="table">
+      <tr>
+        <th>
+          <b>Property Name</b>
+        </th>
+        <th>
+          <b>Default</b>
+        </th>
+        <th>
+          <b>Meaning</b>
+        </th>
+      </tr>
```

Comment:
Can we layout the above html tags similarly with other tables in this doc? E.g.,

```html
<table class="table">
  <tr><th>Property Name</th><th>Meaning</th></tr>
  <tr>
```

Reply:
No problem.